### ISSUE TITLE
How to reproduce the metrics in your paper?

### ISSUE THREAD

**Issue by tiankuan93** [xiaosu-zhu/McQuic#9]
Questions:
I tried different config settings to train the image codec model, and at the lowest quality, I got similar results to the paper. But when using the No.4 model setting, I could not get similar results to the paper. When validating MS-SSIM in the CLIC validation set, the trained model differed from the metrics reported in the paper by about 5-6 dB, which is a very serious difference. The result I reproduced is approximately the red symbol in the figure.
<img width="468" alt="image" src="https://github.com/xiaosu-zhu/McQuic/assets/19601425/adb736f0-bf1d-4886-86ea-2959f67facbb">
I also tried training with an earlier version of cac2921, with the model configuration shown below, and this result still differs from the paper's results by about 5 dB.
<img width="226" alt="image" src="https://github.com/xiaosu-zhu/McQuic/assets/19601425/aa259e69-4b03-41a9-be56-a5891c46dcec">

Request:
I would like to reproduce the metrics in the paper, is it convenient to provide more training details of the model in the paper?
